configure

```diff
@@ -29,6 +30,16 @@ if [ "${INCLUDE_DIR}" ]; then
 	PKG_CXXFLAGS="-I\"${INCLUDE_DIR}\""
 	JASPCOLUMNENCODER_DIR="${INCLUDE_DIR}"
 
+	if git --version 2>&1 >/dev/null; then
+
+		if [ -d "${JASPCOLUMNENCODER_DIR}/.git" ]; then
+			EXTRA_INFO="include_dir, git_repo at:  + $(git --git-dir="${JASPCOLUMNENCODER_DIR}/.git" --work-tree="${JASPCOLUMNENCODER_DIR}" log -1 --oneline)"
+		else
+			EXTRA_INFO='include_dir that is not a git repo.'
```

Comment:
If its not from git we are copying it from somewhere else right?
Maybe print the include path as well for the non-git case?